cardinal/plugins.py

```diff
@@ -184,6 +184,58 @@ def _create_plugin_instance(self, module, config=None):
 
         return instance
 
+    def _register_plugin_events(self, events):
```

Comment:
For consistency, it'd be nice if this took the argument `instance` which is the instance of the plugin (like `_get_plugin_commands`).

Additionally, if you could document the return in the docstring that would be awesome.